src/Avalonia.Themes.Simple/Controls/RefreshContainer.xaml

```diff
@@ -0,0 +1,24 @@
+<ResourceDictionary xmlns="https://github.com/avaloniaui"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <ControlTheme x:Key="{x:Type RefreshContainer}"
+                TargetType="RefreshContainer">
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Grid Background="{DynamicResource ThemeControlMidHighBrush}">
```

Comment:
Why does refresh container need to set a non-transparent background?
You can set transparent background on the ControlTheme, and use TemplateBinding inside.